File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    int i;
    int j;
    do
        //Ask for positive
    {
        n = get_int("Positive number: ");
    }
    //The height of the pyramid
    while (n > 23 || n < 0);
    for (i = 0; i <= n - 1; i++)
    {
        //Print so many spaces
        for (j = 0; j <= n - i - 2; j++)
        {
            printf(" ");
        }
        //Print so many #
        for (j = 0; j <= i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int


while True:
    n = get_int("Positive number: ")
    if 0 <= n <= 23:
        break

# hight
for i in range(n):
    # spaces
    for j in range(n - i - 1):
        print(" ", end="")
    # hashes
    for j in range(i + 2):
        print("#", end="")
    # new line
    print()
